src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx

```diff
@@ -55,6 +55,7 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
 	const [dismissed, setDismissed] = useState(false);
 	const isScrollPlayer = !(dismissed || onScreen);
 	const inputName = 'isDismissed';
+	const relatedContainer: HTMLDivElement | null = document.querySelector('#featured-video__player_related');
```

Comment:
Since we're selecting an element with unique id what do you think of just using `document.getElementById()`? 🤔

Reply:
🤷 There's hardly a difference. I find it more consistent to use the `querySelector` most of  the time instead of switching between different selectors.

Reply:
But there is a difference and it's connected to speed, so maybe it's worth ;-) 

Actually, if there are more than one JWP on page there would be more `.jw-plugin-related` elements, so using `querySelector()` seems like a better idea. But then using the ID in the selector doesn't make sense and we'd need a selector with the player ID and the plugin class, for example: `#featured-video__player-container .jw-plugin-related`.